tests/test_wave_01.py

```diff
@@ -4,7 +4,7 @@
 from viewing_party.party import *
 from tests.test_constants import *
 
-@pytest.mark.skip()
+# @pytest.mark.skip()
```

Comment:
Thank you so much for pointing this out!! I actually ran into an issue regarding removing the decorators here during the course of my project, funnily enough-- every time I removed them (despite them being commented out when I went to remove), they caused the tests to break wholesale. 

Lain and I discussed this (and I think another student encountered the issue as well?), and we guess it might be related to some syntax error on our part, but I managed to replicate this a few times while taking pains to leave everything but the decorators unaltered. In any case, knowing that I was in the right to remove them in the first place leaves me feeling much more confident going forward-- thank you!

Reply:
Hmm, interesting! It might be worth debugging if we have a chance during 1:1 or something. I just gave it a try with a clone of another student's submission who had them commented out and removing them worked OK for me.